### PR TITLE
Filter by posted bounties

### PR DESCRIPTION
**Pull request by brianjcarroll**
First PR! Attempting to implement a second store to hold references to bounties posted by the user (known as userEvents). The store is essentially a copy paste job of eventsStore

### CODE REVIEW

File: app/components/Bounties.tsx [hunk 1]
```diff
@@ -89,14 +103,61 @@ export default function Bounties() {
...
+        setUserEvents(relayUrl, events);
+      }
+    };
+
+    const userPubkey = getUserPublicKey();
+    const filter = userPubkey.length > 0 ? Object.assign(bountyFilter, { authors: [userPubkey] }) : bountyFilter;
+    subscribe([relayUrl], filter, onEvent, onEOSE);
+  };
+
+  const showBounties = (bountyType: keyof typeof BountyType) => {
```
Comment (brianjcarroll) [inline app/components/Bounties.tsx:149]: Acts as a switch for different bounty types (all, userPosted, assigned)

File: app/stores/userEventStore.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import type { Event } from "nostr-tools";
+import { create } from "zustand";
+import { createJSONStorage, devtools, persist } from "zustand/middleware";
+
+interface UserEventStore {
+  userEvents: Record<string, Array<Event>>;
```
Comment (ChristianChiarulli) [inline app/stores/userEventStore.ts:6]: we can keep the user bounty events in the main bounty event store

Comment (brianjcarroll) [reply app/stores/userEventStore.ts]: Done - yep i went back and forth on that. Makes more sense to have them in the same object.

**Review by ChristianChiarulli — APPROVED** [review]
LGTM

**Pull request merged**